blazor/datagrid/how-to/render-grid-inside-tab-with-specific-height.md

```diff
@@ -11,22 +11,34 @@ documentation: ug
 
 By default, DataGrid will occupy the entire space of the parent element when DataGrid [Height](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_Height) and [Width](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_Width) properties are defined as 100%. But if you render the similar DataGrid inside the Tab control, it will consider the entire page and render the DataGrid without horizontal scroller.
 
-To overcome this behavior, render a container element enclosing the DataGrid with specific height and set the DataGrid height as 100%.
+To overcome this behavior, override the below CSS style of the `SfTab` component. This CSS style will hide the default page scroller and add a scroller to the grid to keep the tab header fixed.
+
+```cshtml
+<style>
+    .e-tab > .e-content {
+        height: calc(100% - 36px) !important; /*tab height - tab header height*/
+    }
+    .e-tab > .e-content .e-item {
+        height: 100% !important;
```

Comment:
for this too can we do this without the important

Reply:
yes, without  **!important** works fine.